core/src/main/scala/org/apache/spark/executor/MesosExecutorBackend.scala

```diff
@@ -31,6 +31,9 @@ import org.apache.spark.deploy.SparkHadoopUtil
 import org.apache.spark.scheduler.cluster.mesos.MesosTaskLaunchData
 import org.apache.spark.util.Utils
 
+import scala.collection.mutable
+import scala.util.Random
```

Comment:
There's a specific order of imports for the Spark project. These two should go in the first group, a few lines above, together with `import scala.collection.JavaConverters._`.